package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "dfusion-bot",
+  "version": "0.0.1",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "DEBUG=ERROR-*,WARN-*,INFO-* node src/bot"
+  },
+  "author": "Angel Rodriguez <anxolin@gmail.com> (https://github.com/anxolin)",
+  "license": "ISC",
```

Comment:
Curious as to why this license?

Reply:
This was automatic :( 
I didn't pay to much attention cause I was evaluating.
Actually I usually don't add any author, I'll remove the license and author too